### DISCUSSION TITLE
What is keyword type `normal`?

### DISCUSSION THREAD

**Discussion by ghost** [SpartanJ/ecode#218] [General]
This is the first time I encountered this type of keyword:

https://github.com/SpartanJ/eepp/blob/develop/src/eepp/ui/doc/languages/objeck.cpp#L27

Btw, is there a list of keyword types?

**Comment by SpartanJ** [comment]
It means basically no special highlight, so it's the default font color. [Here's the list](https://github.com/SpartanJ/eepp/blob/develop/src/eepp/ui/doc/syntaxcolorscheme.cpp#L44). I will add documentation later. These names come originally from lite, I kept the naming convention and continue adding some new ones.

**Reply by SpartanJ** [reply]
BTW that pattern can be removed without consequences, IDK why some lite-xl language definitions have it, but it reads as "for any space character use the default font color" which is useless because space characters are invisible.

**Reply by ghost** [reply]
It seems they use `normal` the same way as `symbol`:

https://github.com/lite-xl/lite-xl-plugins/blob/4dd1fe558e5e8abba0039541c25e9e725172222d/plugins/language_cmake.lua

Btw, their cmake lexer is even worse than ecode.

**Reply by ghost** [reply]
https://lite-xl.com/en/tutorials/syntax-highlighting
